edx_proctoring/templates/proctoring/seq_timed_exam_entrance.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+<div class="sequence timed-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+    {{ display_name }} is a Timed Exam ({{total_time}})
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% trans "This exam has a time limit associated with it." %}
+    <strong>
+      {% trans "In order to successfully pass this exam you will have to answer the following questions and problems in the time allotted." %}
+    </strong>
+    {% trans "Once you proceed, you'll start both the exam and the "%} {{total_time|lower}} {% trans " given to you." %}
```

Comment:
"As soon as you indicate that you are ready to start the exam, you will have {{total_time\lower}} to complete the exam."